### ISSUE TITLE
R.once TS type definition miss to context argument and its type 

### ISSUE THREAD

**Issue by jerome-benoit** [selfrefactor/rambda#728]
**Describe the bug**

Rambda type: ```export function once<T extends AnyFunction>(func: T): T;```

And the expected type is more: 

```ts
export const once = <T extends AnyFunction, C = unknown>(
  fn: T,
  context?: C
): T
```

**Context(which version of library)** 
9.1.1

**Comment by selfrefactor** [comment]
Not a bug as Rambda typings are same as Ramda. Still, I will add this and it will be released with next version. I will write once it is done.

**Comment by jerome-benoit** [comment]
The proposed type definition including the complete `once()` function arguments is backward compatible with the previous one. Without it, one can't bound it to another instance if using typescript without `tsc` error, which is a legit use case. 

Thanks.

**Comment by selfrefactor** [comment]
txs for clarification; in this case I will add it and release it.

**Comment by selfrefactor** [comment]
Released with `9.2.0`
